### PR TITLE
speakeasy: setup mintlify integration

### PR DESCRIPTION
**Pull request by simplesagar**
* Creating a combined spec with openapi + codesamples to power mintlify docs
https://www.speakeasyapi.dev/docs/mintlify

### CODE REVIEW

File: .speakeasy/workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+workflowVersion: 1.0.0
+sources:
+    openapi:
+        inputs:
+            - location: ../../apps/web/openapi.json
+        overlays:
+            - location: https://raw.githubusercontent.com/dubinc/dub-node/main/codeSamples.json
+        output: ../../apps/web/openapi_with_sdks.json
```
Comment (ryan-timothy-albert) [inline .speakeasy/workflow.yaml]: output location should be from the root directory, is that correct?

File: .speakeasy/workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+workflowVersion: 1.0.0
+sources:
+    openapi:
+        inputs:
+            - location: ../../apps/web/openapi.json
```
Comment (ryan-timothy-albert) [inline .speakeasy/workflow.yaml]: Same input from root directory

File: .speakeasy/workflow.yaml (hunk 1 above)
Comment (simplesagar) [reply .speakeasy/workflow.yaml]: Thats right this in the `apps` root directory

File: .speakeasy/workflow.yaml (hunk 2 above)
Comment (simplesagar) [reply .speakeasy/workflow.yaml]: Thats right this in the `apps` root directory

**Review by devkiran — APPROVED** [review]

**Pull request closed without merging**